### ISSUE TITLE
user-read-birthdate Scope - Persisting Issue

### ISSUE THREAD

**Issue by liamtrotzuk** [charlie86/spotifyr#111]
Has anyone else noticed a persistent issue with the `user-read-birthdate` scope still appearing when using `spotifyr::scopes`? Using the underlying scopes snippet directly in RStudio produces the correct list of scopes:

```
scopes <- xml2::read_html("https://developer.spotify.com/documentation/general/guides/scopes/") %>%
+   html_nodes('code') %>%
+   html_text() %>%
+   unique()
```
But a different (innaccurate) list of scopes is generated when using `spotifyr::scopes`.  In addition to still generating the `user-read-birthdate` scope, using `spotifyr::scopes` also fails to generate `ugc-image-upload`, `user-read-playback-position`, and `app-remote-control`. Curious whether this is a widespread issue.

**Comment by antaldaniel** [comment]
Can you  please check if your issues is solved with the new CRAN release 2.2.3 or with the development version 2.2.4?

**Comment by liamtrotzuk** [comment]
That issue appears to be fixed with the 2.2.3 CRAN release! Thanks for following up!
